cordova-lib/src/plugman/registry/registry.js

```diff
@@ -236,15 +231,22 @@ function initSettings() {
     module.exports.settings =
     rc('plugman', {
         cache: plugmanCacheDir,
-        registry: 'http://registry.cordova.io',
+        registry: registryURL,
         logstream: fs.createWriteStream(path.resolve(plugmanConfigDir, 'plugman.log')),
         userconfig: path.resolve(plugmanConfigDir, 'config'),
         'cache-min': oneDay
     });
+
+    // if npm is true, use npm registry. 
+    // ~/.plugman/config overides the above config if it exists. 
+    // Need to reset the registry value in settings 
+    if(npm) {
+        settings.registry = 'http://registry.npmjs.org';
```

Comment:
There's no way to avoid this being overwritten in the first place? If not, rather than duplicating the string, can you either:

1. Store this string in a variable and use the variable twice.
2. Simply use the existing registryURL here (since it must be set to the npm registry).